### PR TITLE
feat(multiple):type,pip

### LINKED ISSUE #1: Enable jinad-aws to create multiple JinaD instances (pods) in one vpc
https://github.com/jina-ai/jina/issues/2082
https://github.com/jina-ai/jina-terraform/issues/8

### PR DESCRIPTION
**Pull request by alanthssss**

### CODE REVIEW

**Comment by lusloher** [comment]
Hi, 
These PRs should not end up on the PM project board, as this is for our internal PM workstreams, product development tasks and PRs should be mng. either in the monthly release boards or we can create a separate one for more visibility and clarity :)

**Comment by alanthssss** [comment]
> Hi,
> These PRs should not end up on the PM project board, as this is for our internal PM workstreams, product development tasks and PRs should be mng. either in the monthly release boards or we can create a separate one for more visibility and clarity :)

Fine, I get it :)

File: variables.tf [hunk 1]
```diff
@@ -7,24 +7,45 @@ variable "region" {
...
 
 
-variable "instance_type" {
-  description = <<EOT
-    Mention the ec2 instance type for the JinaD instance
-    EOT
-  type        = string
-  default     = "t2.micro"
-}
+# variable "instance_type" {
```
Comment (deepankarm) [inline variables.tf]: Let's remove the commented lines

Comment (niuzs-alan) [reply variables.tf]: > Hi,
> These PRs should not end up on the PM project board, as this is for our internal PM workstreams, product development tasks and PRs should be mng. either in the monthly release boards or we can create a separate one for more visibility and clarity :)

OK, I get it.

**Comment by niuzs-alan** [comment]
> Hi,
> These PRs should not end up on the PM project board, as this is for our internal PM workstreams, product development tasks and PRs should be mng. either in the monthly release boards or we can create a separate one for more visibility and clarity :)

OK, I get it.

**Review by deepankarm — APPROVED** [review]

**Pull request merged**